Cookbook/Technical-Documents/FastlaneMatch.md

```diff
@@ -54,16 +54,20 @@ We specify individual app identifiers instead of adding all App Store bundle ids
 
 (As of June 2019, `fastlane` could not renew expired profiles automatically or using a command although the documentation mentions that it does)
 - Renew the profile on Apple developer center
-- Run the following command to update central git repository with new profile
+- Run the following command to update match repository with new profile
 
 ```shell
- $ bundle exec fastlane match appstore --team_id {{appstore teamID}} --git_branch {{appstore teamID}} --app_identifier {{appstore bundleID}} --readonly false
+ $ bundle exec fastlane match appstore --team_id {{enterprise/appstore teamID}} --git_branch {{master/appstore teamID}} --app_identifier {{enterprise/appstore bundleID}} --readonly false
 ```
-Note: We specify `app-identifier` so that other valid profiles are not re-generated by mistake
+
+Note: We specify `app-identifier` so that other valid profiles are not re-generated by mistake.
 
 ## Add new device for development
 
-- Run `bundle exec fastlane add_device` locally, it will ask you for the name of the device and its UDID. You can also register your device manually on the portal in the enterprise team account
+- Run `bundle exec fastlane add_device` locally, it will ask you for the name of the device and its UDID, will add it to the developers portal for enterprise team and will automatically regenerate all development provisioning profiles.
```

Comment:
```suggestion
- Run `bundle exec fastlane add_device` locally, it will ask you for the name of the device and its UDID, will add it to the Apple developer center for enterprise team and will automatically regenerate all development provisioning profiles.
```